Add announcement for opening videos in JHelioviewer

dgarciabriseno opened the pull request:
Preview:


![Screen Shot 2023-10-27 at 09 47 05](https://github.com/Helioviewer-Project/helioviewer-project.github.io/assets/94071409/6447769a-8eea-4dee-82a7-e3d6f36388b1)

wafels commented:
You might want to explain what SAMP is, at least briefly. 

"When JHelioviewer is running, it can accept communication using the Simple Application Messaging Protocol, or SAMP. Clicking the button in the helioviewer.org movie dialog sends the details of the movie to JHelioviewer using SAMP."

dgarciabriseno commented:
Added a background section and updated the preview image. Here I added a blurb on SAMP, sampjs, and jhvrequest and a brief technical note that the mp4 itself isn't sent.
